src/ssl.c

```diff
@@ -25393,6 +25540,23 @@ unsigned long wolfSSL_ERR_peek_error(void)
     return wolfSSL_ERR_peek_error_line_data(NULL, NULL, NULL, NULL);
 }
 
+int wolfSSL_ERR_GET_LIB(unsigned long err)
```

Comment:
OpenSSL uses a part of the number to indicate the library.
Can we do this too?

Reply:
I think that it would make recognizing our errors difficult. In my opinion its easier to switch case it here and not worry about OpenSSL <-> wolfSSL error number conversions.